include/zephyr/bluetooth/audio/audio.h

```diff
@@ -1784,6 +1784,58 @@ int bt_audio_unicast_server_register_cb(const struct bt_audio_unicast_server_cb
  */
 int bt_audio_unicast_server_unregister_cb(const struct bt_audio_unicast_server_cb *cb);
 
+/**
+ * @brief Structure holding information of audio stream endpoint
+ */
+struct bt_audio_ep_info {
+	/** The ID of the ASE */
+	uint8_t id;
+	/** The state of the endpoint */
+	uint8_t state;
```

Comment:
We should use `enum bt_audio_state` and move the definitions from `stream.h` to `audio.h`